packages/rolldown/src/rolldown-binding.wasi-browser.js

```diff
@@ -70,53 +70,65 @@ function __napi_rs_initialize_modules(__napiInstance) {
   __napiInstance.exports['__napi_register__transform_9']?.()
   __napiInstance.exports['__napi_register__Bundler_struct_0']?.()
   __napiInstance.exports['__napi_register__Bundler_impl_5']?.()
-  __napiInstance.exports['__napi_register__BindingInputItem_struct_6']?.()
```

Comment:
Why there's changes in napi?

Reply:
I'm also curious that when I run the `just test` command, this file automatically changes.